displayio_cartesian.py

```diff
@@ -81,7 +83,7 @@ class Cartesian(Widget):
 
     .. code-block:: python
 
-        from displayio_cartesian import Cartesian as Plane
+        from adafruit_displayio_layout.widgets.cartesian import Cartesian as Plane
```

Comment:
import here should be `displayio_cartesian`

Reply:
Oh, great catch, I must have missed that!